pkgs/development/python-modules/pyautogui/default.nix

```diff
@@ -0,0 +1,129 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python3Packages
+, pkgs
+, fetchzip
+}:
+let
+  pygetwindow = buildPythonPackage rec {
+    pname = "PyGetWindow";
+    format = "pyproject";
+    version = "0.0.9";
+    doCheck = false;
+    propagatedBuildInputs = [
+      pyrect
+    ];
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "sha256-F4lDVefSswXNgy1xdwg4QBfBaYqQziT29/vwJC3Qpog=";
+    };
+  };
```

Comment:
I am not sure that building those dependencies locally is a good idea. If those dependencies are eventually packaged separately here in `nixpkgs`, it is possible to have a very hard to debug issue where incompatible versions of the dependencies are built on `PYTHONPATH`.

Don't know what other folks think. WDYT @SuperSandro2000 ?

Reply:
good catch! We shouldn't do this in pythonPackages. It can lead to empty outputs or strange version mismatches which can take an hour or two to debug.